src/lang/pt_BR.json

```diff
@@ -0,0 +1,751 @@
+  "Profile.form.locale.label": "Locale",
+  "Profile.form.locale.description": "Localidade do usuário para usar na interface do usuário do MapRoulette.",
+  "Profile.form.leaderboardOptOut.label": "Desativar o placar" ,
+  "Profile.form.leaderboardOptOut.description": "Se sim, você ** não ** aparecerá na tabela de classificação pública.",
+  "Profile.apiKey.header": "Chave de API",
+  "Profile.apiKey.controls.copy.label": "Copiar",
+  "Profile.apiKey.controls.reset.label": "Reset",
+  "Activity.action.updated": "Atualizado",
+  "Activity.action.created": "Criado",
+  "Activity.action.deleted": "Excluído",
+  "Activity.action.taskViewed": "Visto",
+  "Activity.action.taskStatusSet": "Definir status no",
+  "Activity.action.tagAdded": "Tag adicionado para",
+  "Activity.action.tagRemoved": "Tag removido de",
+  "Activity.action.questionAnswered": "Pergunta respondida em",
+  "Activity.item.project": "Projeto",
+  "Activity.item.challenge": "Desafio",
+  "Activity.item.task": "Tarefa",
+  "Activity.item.tag": "Tag",
+  "Activity.item.survey": "Inquérito",
+  "Activity.item.user": "Usuário",
+  "Activity.item.group": "Grupo",
+  "Challenge.basemap.none": "Nenhum",
+  "Admin.Challenge.basemap.none": "Padrão do usuário",
+  "Challenge.basemap.openStreetMap": "OpenStreetMap",
+  "Challenge.basemap.openCycleMap": "OpenCycleMap",
+  "Challenge.basemap.bing": "Bing",
+  "Challenge.basemap.custom": "Personalizado",
+  "Challenge.difficulty.easy": "Fácil",
+  "Challenge.difficulty.normal": "Normal",
+  "Challenge.difficulty.expert": "Especialista",
+  "Challenge.difficulty.any": "Qualquer",
+  "Challenge.keywords.navigation": "Estradas / Pedestres / Ciclovias",
+  "Challenge.keywords.water": "Água",
+  "Challenge.keywords.pointsOfInterest": "Pontos / Áreas de Interesse",
+  "Challenge.keywords.buildings": "Edifícios",
+  "Challenge.keywords.landUse": "Uso da Terra / Limites Administrativos",
+  "Challenge.keywords.transit": "Trânsito",
+  "Challenge.keywords.other": "Outro",
+  "Challenge.keywords.any": "Qualquer coisa",
+  "Challenge.location.nearMe": "Perto de mim",
+  "Challenge.location.withinMapBounds": "Dentro dos limites do mapa",
+  "Challenge.location.intersectingMapBounds": "Interseção de limites do mapa",
+  "Challenge.location.any": "Em qualquer lugar",
+  "Challenge.status.none": "Não aplicável",
+  "Challenge.status.building": "Edifício",
+  "Challenge.status.failed": "Falhou",
+  "Challenge.status.ready": "Pronto",
+  "Challenge.status.partiallyLoaded": "Parcialmente carregado",
+  "Challenge.status.finished": "Terminado",
+  "Challenge.type.challenge": "Desafio",
+  "Challenge.type.survey": "Inquérito",
+  "Editor.none.label": "Nenhum",
+  "Editor.id.label": "Editar no iD (editor da web)",
+  "Editor.josm.label": "Editar no JOSM",
+  "Editor.josmLayer.label": "Editar na nova camada do JOSM",
```

Comment:
trocar "na" por "em uma"

Reply:
Corrigi conforme sugestão! Obrigado! / Correct as per suggestion! Thank you!